scripts/extract_datasets.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+import argparse
+import ast
+import logging
+import os
+
+from .extract_model_names import get_changed_files
```

Comment:
I would probably just copy it over (no need to make the two dependent on eachother for a single function)